style.css

```diff
@@ -188,26 +234,87 @@ html, body {
         width: 100%;
         height: 100%;
         padding: 70px 10px 20px;
-        background-color: #ffffff;
-        transition: 0.3s;
+        background-color: var(--main-bg-color);  
+        transition: 0.5s;
         overflow: auto;
     }
     .header__menu.active {
         top: 0;
     }
     .header__list {
         display: block;
-        padding: 0 0 10px;
+        padding: 0 10px 10px;
     }
     .header__list li {
-        margin: 0 0 10px 10px;
-        border-top: 2px solid #d3d3d3;
-        padding-top: 10px;
-        padding-left: 10px;
+        margin: 0 0 10px;
+        border-top: 2px solid var(--main-border-color);  
+        padding: 7px 10px 0 10px;
+        line-height: 35px;
+    }
+    .header__list li:first-child {
+        border-top: 0px;
     }
     body.lock {
         overflow: hidden;
     }
+    .form-search input {
+        font-size: 12px;
+        width: 90%;
+    }
+    .form-search input[type="image"] {
+        width: 15px;
+        height: 15px;
+    }
+    .header__subnav.active {
+        position: fixed;
+        height: 235px;
+        width: 100%;
+        right: -100%;
+        top: 65px;
+        margin-left: 20px;
+    }
+    .header__subnav li:first-child {
+        display: block;
+        text-transform: uppercase;
+        font-size: 12px;
+        letter-spacing: 1px;
+        text-align: right;
+        height: 43px;
+        line-height: 45px;
+    }
+    .header__sublink#button_back {
+        height: 40px;
+        width: 40px;
+        background-color: var(--main-bg-color);  
+        border: 2px solid var(--main-border-color);  
+        font-size: 20px;
+        float: left;
+    }
+    .header__linkbutton#button_submenu {
+        height: 40px;
+        width: 40px;
+        background-color: var(--main-bg-color);  
+        border: 2px solid var(--main-border-color);  ;
+        font-size: 20px;
+        float: right;
+    }
+    .header__link.active:hover {
+        color: var(--main-text-color);
+    }
+    i {
```

Comment:
why do youapply this to al italic on your page

Reply:
Changed "i" to "div".